packages/ember-testing/lib/setup_for_testing.js

```diff
@@ -29,7 +30,7 @@ export default function setupForTesting() {
   let adapter = getAdapter();
   // if adapter is not manually set default to QUnit
   if (!adapter) {
-    setAdapter(new QUnitAdapter());
+    setAdapter((typeof self.QUnit === 'undefined') ? new Adapter() : new QUnitAdapter());
```

Comment:
hmm, I'm not sure how that would work. In the current state it checks the `QUnit` global at the time of the `setAdapter()` call, but if we put it inside a separate module we would have to do something like `export default (hasQUnit ? QUnitAdapter : Adapter);` and move the check to boot-time, which makes it harder to test.